src/main/java/com/example/gradecalculator/model/UserEditTO.java

```diff
@@ -0,0 +1,17 @@
+package com.example.gradecalculator.model;
+
+public class UserEditTO {
+    private String firstName;
+    private String lastName;
+
+    public UserEditTO() {}
+
+    public String getFirstName() {return firstName;}
+
+    public void setFirstName(String firstName) {this.firstName = firstName;}
+
+    public String getLastName() {return lastName;}
+
+    public void setLastName(String lastName) {this.lastName = lastName;}
```

Comment:
maybe look into lombok and the @Getter and @Setter attributes. This will make the code cleaner IMO